linter/src/main/java/linter/lab3.java

```diff
@@ -0,0 +1,30 @@
+package linter;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Scanner;
+
+public class lab3 {
+    public static void main(String[] args){
+        findErrors("gates.js");
+    }
+
+    public String findErrors(){
+        Scanner s = new Scanner(new File("./src/main/resources/gates.js"));
```

Comment:
Anytime you're bringing in data from another sources (like a file or from an API) it's a good idea to wrap your code in a try catch so that if the file/data doesn't exist, your code knows what to do.